tests/src/test/java/alluxio/client/FileOutStreamIntegrationTest.java

```diff
@@ -111,12 +115,14 @@ public void writeSpecifyLocalTest() throws Exception {
   public void longWriteTest() throws Exception {
     AlluxioURI filePath = new AlluxioURI(PathUtils.uniqPath());
     final int length = 2;
-    FileOutStream os = mFileSystem.createFile(filePath, mWriteUnderStore);
+    FileOutStream os = mFileSystem.createFile(filePath,
+            CreateFileOptions.defaults().setWriteType(WriteType.THROUGH));
     os.write((byte) 0);
     Thread.sleep(Configuration.getInt(Constants.USER_HEARTBEAT_INTERVAL_MS) * 2);
     os.write((byte) 1);
     os.close();
-    checkWrite(filePath, mWriteUnderStore.getUnderStorageType(), length, length);
+    checkWrite(filePath, CreateFileOptions.defaults().setWriteType(WriteType.THROUGH).getUnderStorageType(),
```

Comment:
`CreateFileOptions.defaults().setWriteType(WriteType.THROUGH).getUnderStorageType()` is equivalent to `UnderStorageType.SYNC_PERSIST`